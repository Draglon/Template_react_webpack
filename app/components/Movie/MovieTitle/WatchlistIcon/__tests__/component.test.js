import React from 'react';

import WatchlistIconComponent from '../component';

describe('<WatchlistIconComponent />', () => {
  const props = {
    watchlist: false,
    handleWatchlist: () => {},
  };
  const component = shallow(<WatchlistIconComponent {...props} />);

  it('should match snapshot - watchlist (false)', () => {
    expect(component).toMatchSnapshot();
  });

  it('should match snapshot - watchlist (true)', () => {
    const newProps = {
      ...props,
      watchlist: true,
    };
    component.setProps(newProps);
    expect(component).toMatchSnapshot();
  });
});
