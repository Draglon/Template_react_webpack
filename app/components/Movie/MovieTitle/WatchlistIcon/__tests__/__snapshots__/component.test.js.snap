// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WatchlistIconComponent /> should match snapshot - watchlist (false) 1`] = `
<Icon
  onClick={[Function]}
  type="heart"
/>
`;

exports[`<WatchlistIconComponent /> should match snapshot - watchlist (true) 1`] = `
<Icon
  onClick={[Function]}
  theme="filled"
  type="heart"
/>
`;
